packages/trail-hapi-server/package.json

```diff
@@ -38,16 +38,17 @@
     "coverage": "lab -l -c test",
     "depcheck": "npx depcheck --ignores=config",
     "start": "node start.js",
-    "test": "lab -l test"
+    "test": "lab test --globals __extends,__assign,__rest,__decorate,__param,__metadata,__awaiter,__generator,__exportStar,__values,__read,__spread,__spreadArrays,__await,__asyncGenerator,__asyncDelegator,__asyncValues,__makeTemplateObject,__importStar,__importDefault,__classPrivateFieldGet,__classPrivateFieldSet"
```

Comment:
Why are all of those needed? Can we avoid them?

Reply:
From reading the docs variables flagged as global are ignored when checking for memory leaks.

In master memory leak detection has been disabled using `-l` flag.

This change enables memory leak detection but ignores global variables which would otherwise fail the build:

```
The following leaks were detected:__extends, __assign, __rest, __decorate, __param, __metadata, __awaiter, __generator, __exportStar, __values, __read, __spread, __spreadArrays, __await, __asyncGenerator, __asyncDelegator, __asyncValues, __makeTemplateObject, __importStar, __importDefault, __classPrivateFieldGet, __classPrivateFieldSet
```